.github/workflows/bump_version.yml

```diff
@@ -1,4 +1,4 @@
-name: Release
+name: Bump Version
 on:
   workflow_dispatch:
```

Comment:
Can we [schedule](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule) the workflow to run on a timeline (like every day) so it doesn't have to be done manually?

Reply:
If we schedule version bump, we also need to publish the release regularly because version bump will not work if a version is not published to NPM registry. I think we should have a set policy on how and when we do releases if we schedule it. I will merge as it is for now.